docs/beta/tutorial/get_version.md

```diff
@@ -0,0 +1,59 @@
+# Tutorial: Get Tamr version
+This tutorial will cover basic Python client usage by guiding you through:
+1. Configuring the connection to a Tamr instance
+2. Retrieving the version of that instance
+
+## Prerequisites
+To complete this tutorial you will need access to a Tamr instance.  Specifically, you must have:
```

Comment:
We should also include a link to the Python Client installation instructions here and say that having TC installed is a prereq.